official/docs/ruby/carrier accounts/update_carrier_account.rb

```diff
@@ -0,0 +1,11 @@
+require 'easypost'
+
+EasyPost.api_key = ENV['EASYPOST_API_KEY']
+
+carrier_account = EasyPost::CarrierAccount.retrieve('ca_...')
+carrier_account.description = 'FL Location UPS Account'
+
+carrier_account.credentials['account_number'] = 'B2B2B2'
+carrier_account.save
```

Comment:
Please use the new params for DHLECS, the params getting updated could also be moved to better show what's happening.
```suggestion
carrier_account = EasyPost::CarrierAccount.retrieve('ca_...')

carrier_account.description = 'FL Location UPS Account'
carrier_account.credentials['account_number'] = 'B2B2B2'

carrier_account.save
```